src/main/java/uk/gov/justice/digital/delius/service/ReferenceDataService.java

```diff
@@ -62,4 +68,19 @@ private StandardReference pomAllocationTransferReason(String reason) {
         return standardReferenceRepository.findByCodeAndCodeSetName(reason, POM_ALLOCATION_REASON_DATASET)
                 .orElseThrow(() -> new RuntimeException(String.format("No pom allocation reason found for %s", reason)));
     }
+
+    public Page<KeyValue> getProbationAreasCodes(boolean restrictActive) {
```

Comment:
maybe a bit weird and happy to change - returning Page even though not currently supporting pagination as its friendlier for backwards compatibility reasons to return an object rather than a list, and could potentially add pagination support at a later date?